README.md

```diff
@@ -140,18 +142,18 @@ The validator supports OpenAPI documents in either JSON or YAML format, using th
 .yaml
 .yml 
 ```
-Assuming your command shell supports the use of wildcards, you can use wildcards when specifying the names of files to be validated.
+If the string ends with '/', it will be searched recursively for supported files.
```

Comment:
Is this necessary for the code climate support?

Reply:
Not at all, but a `config.json` passed by codeclimate file looks like this:
```json
{
  "enabled":true,
  "channel":"stable",
  "include_paths": [
    ".gitignore",
    "public/",
    ".gitattributes",
    "LICENSE",
    ".gitlab-ci.yml",
    "Dockerfile",
    "docker-entrypoint.d/",
    ".codeclimate.yml"
  ],
  "debug":"1"
}
```

when executing in [biletado/apidocs](https://gitlab.com/biletado/apidocs/). One can still add a `files` section in `.codeclimate.yml` like I do at the moment for testing until the implementation for the recursive filesearch is implemented. This is than added to the `config.json`

```yaml
version: "2"
plugins:
  openapi:
    enabled: true
    files:
      - public/openapi/jwt.yaml
  csslint:
    enabled: false
  coffeelint:
    enabled: false
  duplication:
    enabled: false
  eslint:
    enabled: false
  fixme:
    enabled: false
  rubocop:
    enabled: false
```